test/integration/openssl.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
@gundalow is correct. Since these tests are run by `ansible-test`, no separate playbook is required.